internal/scheduler/database/executor_repository_test.go

```diff
@@ -61,8 +61,14 @@ func TestExecutorRepository_LoadAndSave(t *testing.T) {
 				}
 				retrievedExecutors, err := repo.GetExecutors(ctx)
 				require.NoError(t, err)
-				executorSort := func(a *schedulerobjects.Executor, b *schedulerobjects.Executor) bool {
-					return a.Id > b.Id
+				executorSort := func(a *schedulerobjects.Executor, b *schedulerobjects.Executor) int {
```

Comment:
We have several uses of `SortFunc` where the sorting is opposite that of what I would think the intuitive one.